gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,380 @@
+
+Attempting to run Flex Gateway with an expired certificate in Connected Mode results in the following:
+
+* New replicas fail to download API configurations from Anypoint Platform, thereby becoming unusable. Existing clusters fail to reload.
+* Logs and metrics fail to upload to Anypoint Platform. Troubleshooting using Anypoint Platform is not possible.
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+Attempting to run Flex Gateway with an expired certificate in Local Mode results in the following:
+
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+If a certificate expires, you can still invoke `flexctl registration renew`, which updates the expired certificate.
+
+To avoid disruptions:
+
+. <<before-you-begin,Download the latest version of Flex Gateway.>>
+. <<renewal-cli-overview,Renew your Flex Gateway registration.>>
+. <<inspection-cli-overview,Verify the success of the renewal process.>>
+
+[[before-you-begin]]
+=== 1. Download Latest Version of Flex Gateway
+
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. You don't need to register and run this version of the gateway.
+
+* xref:flex-install.adoc[]
+
+[[renewal-cli-overview]]
+=== 2. Renew Your Registration
+
+Flex Gateway includes the following CLI command to renew registration:
+
+----
+flexctl registration renew
+----
+
+The command renews an existing Flex Gateway registration with Runtime Manager. For authentication, you run the command with either user credentials or with connected app credentials. Authentication with user credentials requires the `--username` and `--password` flags. Authentication with connected app credentials requires the `--client-id` and `--client-secret` flags.
+
+You can run the following `help` command for information about usage:
+
+[tabs]
+====
+Linux::
++
+[source,ssh]
+----
+flexctl registration renew --help
+----
++
+Use `sudo` if you encounter file permission issues when running this command.
+
+Docker::
++
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command.
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway registration renew --help
+----
+
+Podman::
++
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command.
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway registration renew --help
+----
+====
+
+The `flexctl` CLI provides two methods to renew your registration:
+
+* <<renew-with-user-password,Method 1: Renew Registration with Anypoint Platform User Credentials>>
+* <<renew-with-conn-app,Method 2: Renew Registration with Connected App Credentials>>
+
+[[renew-with-user-password]]
+==== Method 1: Renew Registration with Anypoint Platform User Credentials
```

Comment:
You don't need the Method 1 here.

Just move the intro to the beginning and discuss there. It's obvious that these are two alternatives.

Reply:
The problem with this structure is that you have 4 levels of hierarchy.

Reply:
Fixed!